docs/manual.md

```diff
@@ -444,6 +447,26 @@ command:
 
 `stdout` and `stderr` can be a string or [pattern](#patterns)
 
+### disk-usage
+Validates whether there is enough space remaining on disk.
+
+```yaml
+disk_usage:
+  /:
+    exists: true
+    total_bytes:
+      gt: 1000000000
```

Comment:
The large number of zeros looks like it would be easy to miscount, leading to errors or typos.

Could the value either include an SI unit that is parsed, or be a decimal number of gigabytes?

Reply:
@petemounce unfortunatelly SI units are not supported. But I've used float matchers with exponents, so it's hopefully a bit less error prone.